homeassistant/components/hardkernel/manifest.json

```diff
@@ -5,5 +5,6 @@
   "documentation": "https://www.home-assistant.io/integrations/hardkernel",
   "dependencies": ["hardware", "hassio"],
   "codeowners": ["@home-assistant/core"],
-  "integration_type": "hardware"
+  "integration_type": "hardware",
+  "quality_scale": "internal"
```

Comment:
This would not be internal?

Reply:
I thought it'd be internal as it had no external dependencies and the documentation link is broken.